pkgs/applications/editors/kakoune/plugins/kakoune-search-blocklist

```diff
@@ -0,0 +1,4 @@
+eraserhd/rep
+eraserhd/parinfer-rust
+alexherbo2/kakoune.vim
+kak-lsp/kak-lsp
```

Comment:
Can you sort this alphabetically?
I feel a bit sorry for this nit-pick, but I just like sorted things 😆

Reply:
np, that's a reasonable request, and simple enough in kakoune itself that I should have thought of it originally :sweat_smile: